miasm2/core/graph.py

```diff
@@ -149,3 +159,44 @@ def dot(self):
                                                 self.edge2str(a, b))
         out += "}"
         return out
+
+
+
+    def compute_dominators(self):
+        """Compute dominators of the graph"""
+        nodes = self.nodes()
+        heads = self.heads()
+
+        dominators = defaultdict(set)
+        for n in self.nodes():
```

Comment:
Below, you use `node` as temporary variable. If it is relevant here too, can you develop `n` too ?